string-manipulation-1/src/get-first-char.js

```diff
@@ -0,0 +1,9 @@
+/* exported getFirstChar */
+// storage for the first character in a string.
+// get the first character value of the string and store it.
+// give back the location of stored first character of the string.
+function getFirstChar(string) {
+  var firstChar = [];
```

Comment:
There's no need to make this value start as an empty array, you can just store the value you are retrieving on the line below in the variable as soon as you declare it.